### PR TITLE
add image, input. refactor code

### PR DESCRIPTION
**Pull request by iRohitGaur**

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
Clean good. Good to merge.

**Review by Muzammil-khan710 — APPROVED** [review]
Everything is good and clean !  Good to merge

**Review by itsmebipul00 — APPROVED** [review]

**Comment by itsmebipul00** [comment]
Nice work!!

File: pages/components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+}
+.input_er textarea {
+  border-color: var(--INPUT-W);
+}
+.input_er textarea:focus {
+  border-color: var(--INPUT-W);
+}
+.input_ss textarea {
+  border-color: var(--INPUT-R);
+}
```
Comment (bhushannaikaroskar) [inline pages/components/input/input.css:88]: Rohit can I know why you are directly using tags and not applying class files to these tags and using those class files.

Comment (iRohitGaur) [reply pages/components/input/input.css:88]: I am not using tags directly. I am targeting them in their designated wrappers which have those classes. Gives me more room to work with other elements inside these wrappers.

**Pull request merged**